Formula/ilmbase.rb

```diff
@@ -18,7 +18,7 @@ class Ilmbase < Formula
   keg_only "ilmbase conflicts with `openexr` and `imath`"
 
   # https://github.com/AcademySoftwareFoundation/openexr/pull/929
-  deprecate! date: "2021-04-05", because: :unsupported
+  disable! date: "2022-07-28", because: :unsupported
```

Comment:
Still need to figure out `ctl` and `field3d`. I was originally considering deprecating, but at least `ctl` has recently had commits. No work related to newer `imath`/`openexr` though.